runtime/Dockerfile.opensuse_leap

```diff
@@ -0,0 +1,49 @@
+ARG VERSION_ID
+FROM nvidia/base/opensuse/leap:${VERSION_ID}
+
+# runc dependencies
+RUN zypper install -y \
+        pkg-config \
+        gcc \
+        libseccomp-devel \
+        libapparmor-devel \
+        libselinux-devel && \
+    rm -rf /var/cache/zypp/*
+
+RUN go get github.com/LK4D4/vndr
+
+# runc
+WORKDIR $GOPATH/src/github.com/opencontainers/runc
```

Comment:
Can you rebase, this Dockerfile becomes a lot simpler :) !

Reply:
done